pkgs/development/compilers/neko/default.nix

```diff
@@ -13,6 +13,13 @@ stdenv.mkDerivation rec {
     sha256 = "19rc59cx7qqhcqlb0znwbnwbg04c1yq6xmvrwm1xi46k3vxa957g";
   };
 
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/HaxeFoundation/neko/commit/ff5da9b0e96cc0eabc44ad2c10b7a92623ba49ee.patch";
```

Comment:
Please reference the origin of that patch. It is not on any branch on the upstream repository.

Reply:
Added in comment.